core/src/test/java/io/grpc/internal/RetriableStreamTest.java

```diff
@@ -147,6 +147,27 @@ public double nextDouble() {
   private final ChannelBufferMeter channelBufferUsed = new ChannelBufferMeter();
   private final FakeClock fakeClock = new FakeClock();
 
+  private static double calculateJitterFactor() {
+    return (0.8 + FAKE_RANDOM * 0.4);
+  }
+
+  private static long calculateInitialBackoff() {
+    return (long) (INITIAL_BACKOFF_IN_SECONDS * calculateJitterFactor());
+  }
+
+  private static long calculateBackoff() {
```

Comment:
I'd suggest the name `calculateSecondBackoff`, but feel free to ignore this advice.
Another approach would be `calculateBackoff(int priorBackoffs)` where instead of the 3 methods you use 0, 1, 2.  Then have a for loop that does the multiplication.